### PR TITLE
bugfix(dmn): Allow DMN with Legacy Namespaces to be deployed

### LINKED ISSUE #194: DMN Diagrams from C7 Modeler are not deployable
**Describe the bug**
DMN Diagrams from C7 Modeler are not deployable

**To Reproduce**
1. Create an empty DMN Diagram in the Modeler
2. Set TTL
3. Try to deploy
4. Get Error: ENGINE-22004 Unable to transform DMN resource 'diagram_1.dmn'. [ deploy-error ]

**Expected behavior**
DMN Diagrams should be deployable like BPMN Diagrams

### PR DESCRIPTION
**Pull request by javahippie**
A draft to keep the state of this issue transparent

Fixes #194

### CODE REVIEW

**Comment by javahippie** [comment]
Still a draft for two reasons:
* Considering if the test coverage can be increased by meaningful tests
* Seeing some other tests for the XML parser which I could extend with the fallback namespace

**Comment by javahippie** [comment]
Created some more diffs than this issue is concerned with, as I tried to "leave the place cleaner than I found it". I fixed some constant naming, adjusted some code to use a more modern code Style (lambdas, etc.)

**Comment by javahippie** [comment]
As this changes engine behavior, I'd be really happy about some more eyes for a review @kthoms @hauptmedia

**Comment by hauptmedia** [comment]
Will checkout this branch and do some test/review later today

File: model-api/bpmn-model/src/main/java/org/operaton/bpm/model/bpmn/impl/instance/operaton/OperatonErrorEventDefinitionImpl.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
 import org.operaton.bpm.model.xml.impl.instance.ModelTypeInstanceContext;
 import org.operaton.bpm.model.xml.type.ModelElementTypeBuilder;
 import org.operaton.bpm.model.xml.type.attribute.Attribute;
-import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.CAMUNDA_ATTRIBUTE_EXPRESSION;
+import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.OPERATON_ATTRIBUTE_EXPRESSION;
 import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.CAMUNDA_ELEMENT_ERROR_EVENT_DEFINITION;
```
Comment (a-m-zill) [inline model-api/bpmn-model/src/main/java/org/operaton/bpm/model/bpmn/impl/instance/operaton/OperatonErrorEventDefinitionImpl.java:26]: @Tim, has you missed here one constant?

File: model-api/bpmn-model/src/main/java/org/operaton/bpm/model/bpmn/impl/instance/operaton/OperatonTaskListenerImpl.java [hunk 2]
```diff
@@ -31,10 +31,10 @@
...
 
-import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.CAMUNDA_ATTRIBUTE_CLASS;
-import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.CAMUNDA_ATTRIBUTE_DELEGATE_EXPRESSION;
-import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.CAMUNDA_ATTRIBUTE_EVENT;
-import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.CAMUNDA_ATTRIBUTE_EXPRESSION;
+import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.OPERATON_ATTRIBUTE_CLASS;
+import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.OPERATON_ATTRIBUTE_DELEGATE_EXPRESSION;
+import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.OPERATON_ATTRIBUTE_EVENT;
+import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.OPERATON_ATTRIBUTE_EXPRESSION;
 import static org.operaton.bpm.model.bpmn.impl.BpmnModelConstants.CAMUNDA_ELEMENT_TASK_LISTENER;
```
Comment (a-m-zill) [inline model-api/bpmn-model/src/main/java/org/operaton/bpm/model/bpmn/impl/instance/operaton/OperatonTaskListenerImpl.java:38]: @javahippie here also. Is that intentional?

**Comment by hauptmedia** [comment]
I can confirm that after applying this patch you can deploy DMN Models from the C7 Modeler again.

**Review by a-m-zill — APPROVED** [review]
👍🏼 LGTM and Thx for your work @javahippie

**Comment by hauptmedia** [comment]
Please always use the squash option to merge pull requests, so it's easier to follow changes on the main branch

**Comment by javahippie** [comment]
No reason to squash here, as I amended and force pushed all of my commits ;)

**Pull request merged**